bin/grouper-api

```diff
@@ -29,6 +31,70 @@ class Application(tornado.web.Application):
         super(Application, self).__init__(*args, **kwargs)
 
 
+class SendEmailThread(Thread):
+    """Background thread for sending emails from the async pool
+
+    This class thread will exist on multiple servers in a standard Grouper production environment
+    so we need to ensure that it's race-safe.
+    """
+
+    def send_emails(self, session, now_ts, dry_run=False):
```

Comment:
It makes sense to run the email-sending thread in the grouper api server(s), but the DB-updating send_emails method should be importable, probably in grouper.util?